packages/x-data-grid-pro/src/typeOverloads/modules.ts

```diff
@@ -44,6 +45,17 @@ export interface GridEventLookupPro {
    * Fired when a new batch of rows is requested to be loaded. Called with a [[GridFetchRowsParams]] object.
    */
   fetchRows: { params: GridFetchRowsParams };
+  // Data source
+  /**
+   * Fired when the grid needs to fetch a new batch of rows from the data source.
+   * @ignore - do not document.
+   */
+  getRows: { params: GridGetRowsParams };
+  /**
+   * Fired when the new data is successfully added to the grid either directly from the data source or from the cache
+   * @ignore - do not document.
+   */
+  rowsFetched: {};
```

Comment:
How about prefixing data source events for clarity? E.g. `dataSource.getRows`, `dataSource.rowsFetched`

Reply:
Events removed, since https://github.com/mui/mui-x/pull/13878/commits/6ef5bf6023456a83cf6a55fc8ed6d4a768ac7277